recipes/cvxpy/meta.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: cvxpy
+  version: "0.4.2"
+
+source:
+  git_url: https://github.com/cvxgrp/cvxpy
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - cvxpy = cvxpy:main
+    #
+    # Would create an entry point called cvxpy that calls cvxpy.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+build:
+  noarch_python: True
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - setuptools
+    - six
+    - toolz
+    - cvxcanon >=0.0.22
+    - fastcache
+    - python
+    - ecos >=2
+    - scs >=1.1.3
+    - multiprocess
+    - numpy >=1.9
+    - scipy >=0.15
+
+
+  run:
+    - python
+    - ecos >=2
+    - scs >=1.1.3
+    - multiprocess
+    - fastcache
+    - six
+    - toolz
+    - numpy >=1.9
+    - scipy >=0.15
+    - cvxcanon >=0.0.22
+
+test:
+  # Python imports
+  imports:
+    - cvxpy
+    - cvxpy.atoms
+    - cvxpy.atoms.affine
+    - cvxpy.atoms.elementwise
+    - cvxpy.constraints
+    - cvxpy.expressions
+    - cvxpy.expressions.constants
+    - cvxpy.expressions.variables
+    - cvxpy.interface
+    - cvxpy.interface.numpy_interface
+    - cvxpy.lin_ops
+    - cvxpy.problems
+    - cvxpy.problems.problem_data
+    - cvxpy.problems.solvers
+    - cvxpy.tests
+    - cvxpy.transforms
+    - cvxpy.utilities
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/cvxgrp/cvxpy/
+  license: GPLv3
```

Comment:
Please ask upstream to add a license file. Also please add that license file to `MANIFEST.in` to ensure it is included in `sdist`s and related packages.